spec/jobs/mail_alias_sync_job_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+require Rails.root.join('spec', 'support', 'mocks', 'fake_http')
+
+RSpec.describe MailAliasSyncJob, type: :job do
+  describe '#perform' do
+    let(:job) { described_class.new }
+    let(:mail_alias) do
+      FactoryBot.create(:mail_alias, :with_user, email: 'test@sandbox86621.eu.mailgun.org')
```

Comment:
Why are we using a mailgun sandbox email address here?

Reply:
Because we still use that everywhere. It should be removed later on, but we do not have a new sandbox (maybe we can create one ourself or just don't care and use example.com)

Reply:
I haven't checked the other occurrences, but in this case, using example.com would suffice I think.